.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: go
```

Comment:
Not something we have to solve with this PR but I wonder if we should templatize this file somehow so we don't have to change all of them each time we want to support different versions of Go for CI.

It could even be something like when running in CI it checks whether the versions of Go match the "master" version in some shared CI repo or something and fails the test run if it it's not up to date.

Let's forget doing anything for this repo, but wanted to bring it up.  Let's find a way to solve this in the near future.